aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -1465,13 +1465,17 @@ The rules for determining a binding source from a parameter:
     1. Header: [`[FromHeader]`](xref:Microsoft.AspNetCore.Mvc.FromHeaderAttribute)
     1. Body: [`[FromBody]`](xref:Microsoft.AspNetCore.Mvc.FromBodyAttribute)
     1. Service: [`[FromServices]`](xref:Microsoft.AspNetCore.Mvc.FromServicesAttribute)
-    1. Parameter values: [`[AsParameters]](<xref:Microsoft.AspNetCore.Http.AsParametersAttribute)
+    1. Parameter values: [`[AsParameters]`](<xref:Microsoft.AspNetCore.Http.AsParametersAttribute)
 1. Special types
-    1. `HttpContext`
-    1. `HttpRequest`
-    1. `HttpResponse`
-    1. `ClaimsPrincipal`
-    1. `CancellationToken`
+    1. [`HttpContext`](xref:Microsoft.AspNetCore.Http.HttpContext)
+    1. [`HttpRequest`](xref:Microsoft.AspNetCore.Http.HttpContext.Request%2A)
+    1. [`HttpResponse`](xref:Microsoft.AspNetCore.Http.HttpContext.Response%2A)
+    1. [`ClaimsPrincipal`](xref:Microsoft.AspNetCore.Http.HttpContext.User%2A)
+    1. [`CancellationToken`](xref:Micrsoft.AspNetCore.Http.HttpContext.RequestAborted%2A)
+    1. [`IFormFileCollection`](xref:Microsoft.AspNetCore.Http.IFormFileCollection.Files%2A)
+    1. [`IFormFile`](xref:Microsoft.AspNetCore.Http.IFormFile)
+    1. [`Stream`](xref:Microsoft.AspNetCore.HttpRequest.Body%2A)
```

Comment:
```suggestion
    1. [`Stream`](ref:Microsoft.AspNetCore.Http.HttpRequest.Body)
```

Reply:
This is supposed to stay an `xref`, right?

Reply:
oops, fixed